CA1008: Compile error with DevDiv WorkItem reference

#### Analyzer package
Microsoft.ApiDesignGuidelines.Analyzers.UnitTests

#### Analyzer
CA1008

#### Repro steps
[A test](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/EnumsShouldHaveZeroValueTests.cs#L24) contains the following code:
```csharp
[System.Flags]
public enum E4
{
    A4 = 0,
    B4 = (uint)2  // Not a constant
}
```
The line with comment results in:
```
error CS0266: Cannot implicitly convert type 'uint' to 'int'. An explicit conversion exists (are you missing a cast?)
```

This test is decorated with DevDiv WorkItem ID 836193. Can someone from within Microsoft check if this explains the comment? I would like to know if this compile error is intentional or a mistake. If intentional, I would like to add a descriptive comment about what is going on here.

--- a/src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/EnumsShouldHaveZeroValueTests.Fixer.cs
+++ b/src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/EnumsShouldHaveZeroValueTests.Fixer.cs
@@ -30,8 +30,7 @@ namespace Microsoft.ApiDesignGuidelines.Analyzers.UnitTests
             return new CSharpEnumsShouldHaveZeroValueFixer();
         }
 
-        [WorkItem(836193, "DevDiv")]
-        [Fact(Skip = "https://github.com/dotnet/roslyn-analyzers/issues/1032")]
+        [Fact]
         public void CSharp_EnumsShouldZeroValueFlagsRename()
         {
             var code = @"
@@ -63,7 +62,7 @@ public enum E3
 public enum E4
 {
     A4 = 0,
-    B4 = (uint)2  // Not a constant
+    B4 = (int)2  // Sample comment
 }
 
 [System.Flags]
@@ -102,7 +101,7 @@ public enum E3
 public enum E4
 {
     None = 0,
-    B4 = (uint)2  // Not a constant
+    B4 = (int)2  // Sample comment
 }
 
 [System.Flags]

--- a/src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/EnumsShouldHaveZeroValueTests.cs
+++ b/src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/EnumsShouldHaveZeroValueTests.cs
@@ -19,8 +19,7 @@ namespace Microsoft.ApiDesignGuidelines.Analyzers.UnitTests
             return new EnumsShouldHaveZeroValueAnalyzer();
         }
 
-        [WorkItem(836193, "DevDiv")]
-        [Fact(Skip = "https://github.com/dotnet/roslyn-analyzers/issues/1032")]
+        [Fact]
         public void CSharp_EnumsShouldZeroValueFlagsRename()
         {
             // In enum '{0}', change the name of '{1}' to 'None'.
@@ -58,7 +57,7 @@ public enum E3
 public enum E4
 {
     A4 = 0,
-    B4 = (uint)2  // Not a constant
+    B4 = (int)2  // Sample comment
 }
 
 [System.Flags]

